src/main/resources/templates/db_scripts/migration_scripts/create_default_job_profile.sql

```diff
@@ -0,0 +1,18 @@
+INSERT INTO
```

Comment:
These are not migration scripts , so let's move it to a common folder

Reply:
moved to the "default_data" folder